### PR TITLE
Table Inline Select onBlur Bug

### PR DESCRIPTION
**Pull request by nickwmurphy**
### Summary
- Fixed a bug where the inline select would not close on blur for React 15 clients

### CODE REVIEW

**Comment by dancingshell** [comment]
version bump + readme?

**Comment by nickwmurphy** [comment]
Just got those done!

File: table/README.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ A table component with checkboxes, sticky head, sticky right panel, and horizont
 ![table](https://user-images.githubusercontent.com/8366399/34239862-2189eda6-e5bf-11e7-8254-b4e2d94ccdf4.gif)
 # Change Log
 
+##### 12.4.1
+- Fixed a bug where the inline select would not close on blur
```
Comment (dancingshell) [inline table/README.md]: might mention that it was only for react 15 clients

File: table/src/InlinePortal.jsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+/* eslint-disable import/no-extraneous-dependencies */
 import ReactDOM from 'react-dom';
+/* eslint-enable import/no-extraneous-dependencies */
```
Comment (dancingshell) [inline table/src/InlinePortal.jsx]: I don't know why it says that, its clearly being used 🤷‍♀️

Comment (nickwmurphy) [reply table/src/InlinePortal.jsx]: 🤷‍♂️

**Pull request merged**